frontend/website-redesign/src/components/AboutpageRows.re

```diff
@@ -0,0 +1,217 @@
+        letterSpacing(`px(-1)),
+        marginBottom(`rem(1.5)),
+      ]),
+    ]);
+  let heroRowImage =
+    style([
+      width(`rem(21.)),
+      marginTop(`rem(1.)),
+      media(
+        Theme.MediaQuery.desktop,
+        [marginTop(`zero), height(`rem(38.5)), width(`rem(38.5))],
+      ),
+    ]);
+  let firstImage =
+    merge([
+      heroRowImage,
+      style([
+        media(
+          Theme.MediaQuery.tablet,
+          [position(`absolute), right(`zero)],
+        ),
+      ]),
+    ]);
+  let secondImage =
+    merge([
+      heroRowImage,
+      style([
+        backgroundImage(`url("/static/img/triangle_mobile.png")),
+        media(
+          Theme.MediaQuery.tablet,
+          [
+            position(`absolute),
+            left(`zero),
+            backgroundImage(`url("/static/img/triangle_tablet.png")),
+          ],
+        ),
+        media(
+          Theme.MediaQuery.desktop,
+          [backgroundImage(`url("/static/img/triangle_desktop.png"))],
+        ),
+      ]),
+    ]);
+  let imageContainer = style([position(`relative)]);
+
+  let copy = Theme.Type.paragraph;
+  let rule =
+    style([
+      width(`percent(100.)),
+      color(Theme.Colors.digitalBlack),
+      border(`px(1), `solid, black),
+    ]);
+  let orange =
+    merge([copy, style([display(`inlineBlock), color(orange)])]);
+};
+
+[@react.component]
+let make = () => {
+  <div className=Styles.rowBackgroundImage>
+    <div className=Styles.container>
+      <div className=Styles.firstColumn>
+        <hr className=Styles.rule />
+        <h2 className=Styles.header>
+          {React.string("It's Time to Own Our Future")}
+        </h2>
+        <p className=Styles.subhead>
+          {React.string(
+             "Living in today's world requires giving up a lot of control.",
+           )}
+        </p>
+        <p className=Styles.copy>
+          {React.string(
+             "Every day, we give up control of intimate data to large tech companies to use online services. We give up control of our finances to banks and unaccountable credit bureaus. We give up control of our elections to voting system companies who run opaque and unauditable elections. ",
+           )}
+        </p>
+        <Spacer height=1.75 />
+        <p className=Styles.copy>
+          {React.string(
+             "Even when we try to escape this power imbalance and participate in blockchains, most of us give up control and trust to third parties to verify transactions. Why? Because running a full node requires expensive hardware, unsustainable amounts of electricity and tons of time to sync increasingly heavier and heavier chains.",
+           )}
+        </p>
+        <Spacer height=1.75 />
+        <p className=Styles.copy>
+          <strong>
+            {React.string("But it doesn't have to be this way. ")}
+          </strong>
+        </p>
+      </div>
+      <img
+        className=Styles.firstImage
+        src="/static/img/02_About_2_1232x1232.jpg"
```

Comment:
`Nit`: it would be awesome if we can change these filenames to something more meaningful.

Reply:
gotcha!